pages/vi/profiles/RheuX.md

```diff
@@ -0,0 +1,45 @@
+# Hello 👋 
+
+## Basic Info
+### My name is Axel Nicholas Lorens
```

Comment:
Also please see https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/3383#discussion_r1793841981

Reply:
Hi, I did some changes with the semantics, I only use it when necessary, if I need to bring a new topic.

Also, I'm not sure what you mean by `include the OS`, Can you see the OS that I put in my previous and current .md? I put down a Macbook and Windows 11 for my OS. Unless you're referring to another OS term that I'm not sure about

Reply:
> Hi, I did some changes with the semantics, I only use it when necessary, if I need to bring a new topic.
> 
> Also, I'm not sure what you mean by `include the OS`, Can you see the OS that I put in my previous and current .md? I put down a Macbook and Windows 11 for my OS. Unless you're referring to another OS term that I'm not sure about

I apologize; I missed that you already mentioned your OS in the document. It typically appears as a list early in the document, which is probably why I overlooked it.